src/app/docs/[...slug]/page.jsx

```diff
@@ -40,14 +40,14 @@ export async function generateMetadata({ params }) {
   if (!post) return notFound();
 
   const {
-    data: { title, ogImage },
+    data: { title },
     excerpt,
   } = post;
 
   return getMetadata({
     title: `${title} - Neon Docs`,
     description: isReleaseNotes ? 'The latest product updates from Neon' : excerpt,
-    imagePath: ogImage,
+    imagePath: `https://neon-next-git-og-image-neondatabase.vercel.app/api/og-image?title=${title}`,
```

Comment:
Let's use vercel system env variables to have ability to test og images in preview and non preview environments.

It feels like we can use `VERCEL_ENV` to check whether it's preview or not preview environment. If it's preview we can build a url for it using `VERCEL_BRANCH_URL` , if not we can use our variable with primary domain.
https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables